### DISCUSSION TITLE
AttachAction recordSelect getOptionLabelFromRecordUsing appears to be ignored

### DISCUSSION THREAD

**Discussion by mmccall10** [filamentphp/filament#12345] [Help] [answered]
### Package

Panel builder

### Package Version

3.2.63

### How can we help you?

I'm not sure if I'm missing something but I can't get a custom dropdown to work when using the `AttachAction` in a relationship manager. Any help greatly appreciated. 

Based on the [docs](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#customizing-the-select-field-in-the-associate-modal) for select and other working examples would expect something like this to work. However, it appears to be ignored.

```php
Tables\Actions\AttachAction::make()
                    ->recordSelect(
                        fn (Select $select) => $select->getOptionLabelFromRecordUsing(function ($record) {
                            return $record->name.':  '.$record->purpose;
                        }),
                    ),
```

**Comment by leandrocfe** [comment, marked as answer]
```php
->recordTitle(fn(Model $record): string => $record->name . ': ' . $record->purpose)
```

**Reply by mmccall10** [reply]
Oh awesome, Thank you!
